homeassistant/components/hue/v1/device_trigger.py

```diff
@@ -162,15 +179,17 @@ async def async_attach_trigger(bridge, device_entry, config, action, automation_
 
 
 @callback
-def async_get_triggers(bridge: "HueBridge", device: DeviceEntry):
+def async_get_triggers(
+    bridge: "HueBridge", device: DeviceEntry
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>

Reply:
I have reworked this to remove the incorrect GetAutomation*** aliases